init/sysinit.go

```diff
@@ -157,6 +159,21 @@ func SysInit() error {
 		},
 		func(cfg *config.Config) error {
 			log.Infof("RancherOS %s started", config.VERSION)
+			ifaces, err := net.Interfaces()
```

Comment:
Please move this to it's own function like printStartupStatus().  This is too much code to embed inline